app/Services/DataImporterService.php

```diff
@@ -63,6 +64,7 @@ public function run(string $whoRunsIt = "admin"): void
             new WindDataImporterJob($this->importInfoId),
             new WheeMoveDataImporterJob($this->importInfoId),
             new ZwingsDataImporterJob($this->importInfoId),
+            new HopDataImporterJob($this->importInfoId),
```

Comment:
Do we care about alphabetical order?

Reply:
In ProviderSeeder it would be nice to keep it in alphabetical order but in DataImporterService providers should be ordered by the largest number of cities with assigned countries - e.g. Bolt and Tier should go first, surely. It needs research which configuration scrap the most locations.

Reply:
Rearranged